samples/core/tutorials/estimators/linear.ipynb

```diff
@@ -17,7 +23,8 @@
       "display_name": "Python 3",
       "language": "python",
       "name": "python3"
-    }
+    },
+    "accelerator": "GPU"
```

Comment:
Did you mean to turn this on?

Reply:
Not getting a performance improvement. Removed.